packages/wallet-sdk/src/CoinbaseWalletProvider.ts

```diff
@@ -153,16 +152,14 @@ export class CoinbaseWalletProvider extends EventEmitter implements ProviderInte
   readonly isCoinbaseWallet = true;
 
   protected readonly updateListener = {
-    onAccountsUpdate: ({ accounts, source }: AccountsUpdate) => {
+    onAccountsUpdate: (accounts: AddressString[]) => {
       if (areAddressArraysEqual(this.accounts, accounts)) return;
       this.accounts = accounts;
```

Comment:
provider won't be in charge of storing accounts anymore in upcoming PR https://github.com/coinbase/coinbase-wallet-sdk/pull/1331